### PR TITLE
Translate docs/helm/helm_get in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>

Translate docs/helm/helm_get in Korean
- https://helm.sh/docs/helm/helm_get/

### CODE REVIEW

**Comment by bluefriday** [comment]
@jmnote @jmyung 리뷰 요청 드립니다.

File: content/ko/docs/helm/helm_get_hooks.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:36]:
```suggestion
      --kube-token string                인증에 사용할 베어러(bearer) 토큰
```

File: content/ko/docs/helm/helm_get_notes.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:34]:
```suggestion
      --kube-token string                인증에 사용할 베어러(bearer) 토큰
```

File: content/ko/docs/helm/helm_get_values.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:23]:
```suggestion
  -h, --help            values 에 대한 도움말
```

File: content/ko/docs/helm/helm_get_notes.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:44]:
```suggestion
      --repository-config string         저장소 이름 및 URL을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
```

File: content/ko/docs/helm/helm_get_values.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:46]:
```suggestion
      --repository-config string         저장소 이름 및 URL을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
```

File: content/ko/docs/helm/helm_get.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get.md:48]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_all.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_all.md:49]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_hooks.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:49]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_manifest.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:51]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_notes.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:47]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get_values.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:49]:
```suggestion
      --stderrthreshold severity         이 기준점 이상일 때 표준오류(stderr)로 로그가 출력된다 (기본값 2)
```

File: content/ko/docs/helm/helm_get.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get.md:49]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_all.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_all.md:50]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_hooks.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:50]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_manifest.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:52]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_notes.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:48]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_values.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:50]:
```suggestion
  -v, --v Level                          로그 상세표시 레벨
```

File: content/ko/docs/helm/helm_get_hooks.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:14]:
```suggestion
훅은 YAML로 형식이며 YAML '---\n' 구분자로 구분된다.
```

File: content/ko/docs/helm/helm_get_hooks.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:12]:
```suggestion
이 명령어는 지정한 릴리스에 대한 훅을 다운로드한다.
```

File: content/ko/docs/helm/helm_get_manifest.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:12]:
```suggestion
이 명령어는 지정한 릴리스에 대해 생성된 매니페스트를 가져온다.
```

File: content/ko/docs/helm/helm_get_values.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:24]:
```suggestion
  -o, --output format   지정한 형식으로 출력을 표시. 허용되는 값: table, json, yaml (기본값 table)
```

File: content/ko/docs/helm/helm_get_hooks.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_hooks.md:25]:
```suggestion
      --revision int   리비전으로 명명된 릴리스 가져오기
```

File: content/ko/docs/helm/helm_get_manifest.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_manifest.md:27]:
```suggestion
      --revision int   리비전으로 명명된 릴리스 가져오기
```

File: content/ko/docs/helm/helm_get_notes.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_notes.md:23]:
```suggestion
      --revision int   리비전으로 명명된 릴리스 가져오기
```

File: content/ko/docs/helm/helm_get_values.md
Comment (jmnote) [inline content/ko/docs/helm/helm_get_values.md:25]:
```suggestion
      --revision int    리비전으로 명명된 릴리스 가져오기
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다.

**Comment by bluefriday** [comment]
> 수고하셨습니다.

@jmnote 리뷰 감사드립니다.

**Comment by jmnote** [comment]
lgtm

File: content/ko/docs/helm/helm_get.md
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get.md:44]: Are you sure about this file path change? Or is this an accident?

https://github.com/helm/helm-www/blame/master/content/en/docs/helm/helm_get.md#L44 says:

 --repository-cache string          path to the file containing cached repository indexes (default "~/.cache/helm/repository")

File: content/ko/docs/helm/helm_get_all.md
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_all.md:45]: Are you sure about this file path change? Or is this an accident?

https://github.com/helm/helm-www/blame/master/content/en/docs/helm/helm_get_all.md#L45 says:

 --repository-cache string          path to the file containing cached repository indexes (default "~/.cache/helm/repository")

File: content/ko/docs/helm/helm_get_hooks.md
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_hooks.md:45]: Please check this default path (see other comments).

File: content/ko/docs/helm/helm_get_manifest.md
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_manifest.md:47]: Please check this default path (see other comments).

File: content/ko/docs/helm/helm_get_notes.md
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_notes.md:43]: Please check this default path (see other comments).

File: content/ko/docs/helm/helm_get_values.md
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_get_values.md:45]: Please check this default path (see other comments).

**Comment by bridgetkromhout** [comment]
This LGTM once we fix the file path (which I think changed accidentally?)

File: content/ko/docs/helm/helm_get.md
Comment (bluefriday) [reply content/ko/docs/helm/helm_get.md:44]: It's my mistake. 
In previous I translated 'helm' command  (https://github.com/helm/helm-www/pull/802).
And in that docs, 'repository cache string path' was set to '~/snap/code/common/.cache/helm/repository'.
After the PR was merged, I used the same sentence for uniformity in translation.

But I couldn't confirm that only that path is different from other commands.
I'll fix that in every commit I've PRed. Thank you.

File: content/ko/docs/helm/helm_get_all.md
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_all.md:45]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_hooks.md
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_hooks.md:45]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_manifest.md
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_manifest.md:47]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_notes.md
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_notes.md:43]: I will modify the same as above.

File: content/ko/docs/helm/helm_get_values.md
Comment (bluefriday) [reply content/ko/docs/helm/helm_get_values.md:45]: I will modify the same as above.

**Comment by bluefriday** [comment]
> This LGTM once we fix the file path (which I think changed accidentally?)

@bridgetkromhout  I modified it according to contents you reviewed.

**Review by bridgetkromhout — APPROVED** [review]
LGTM

**Pull request merged**
